ui/src/containers/organisations.list/settings/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { Flex, Separator, Switch, Text } from "@raystack/apsara";
+import { V1Beta1Preference } from "@raystack/frontier";
+import { useFrontier } from "@raystack/frontier/react";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { useParams } from "react-router-dom";
+import PageHeader from "~/components/page-header";
+
+export default function OrgSettingPage() {
+  let { organisationId } = useParams();
+  const [socialLogin, setSocialLogin] = useState<boolean>(false);
+  const [mailLink, setMailLink] = useState<boolean>(false);
+
+  const [preferences, setPreferences] = useState<V1Beta1Preference[]>([]);
+  const { client, activeOrganization: organization } = useFrontier();
+
+  const fetchOrganizationPreferences = useCallback(async () => {
+    const {
+      // @ts-ignore
+      data: { preferences },
+    } = await client?.frontierServiceListOrganizationPreferences(
+      organisationId as string
+    );
+
+    setPreferences(preferences);
+  }, [client, organisationId]);
+
+  useEffect(() => {
+    if (organisationId) fetchOrganizationPreferences();
+  }, [organisationId, client, fetchOrganizationPreferences]);
+
+  const preferencesMap = useMemo(() => {
+    return preferences.reduce<Record<string, Record<string, string>>>(
+      (map, el) => {
+        // @ts-ignore
+        map[el.name] = el;
+        return map;
+      },
+      {}
+    );
+  }, [preferences]);
+
+  useEffect(() => {
+    if (preferencesMap["social_login"])
```

Comment:
The auth strategies are global level in the backend, not org level.
this section should be on a root level setting page

Reply:
we are using same for SDK as well